docs/developer/creating-a-plugin.rst

```diff
@@ -311,40 +217,31 @@ This submodule should contain all the constants that can be later imported in pr
     [. . .]
 
 Convert
-~~~~~~~
+^^^^^^^
+
 This submodule might contain two files: ``convert.py`` with all functions used for conversion between
 base classes and programs inside the plugin; ``dataimport.py`` with import classes that are used in
```

Comment:
I think dataimport.py will not be dynamically discovered by the imports. Probably we should not mention it. @delarosatrevin ?

Reply:
Nope, it has been moved into the convert module, so we have some dataimport.py in some plugins, but it is not really relevant...internally to a submodule, it can be structured in any way.

Reply:
So, if I have understood you correctly, it won't be present in general since it has been moved to the convert module, but at the same time it is present in some plugins. Should the doc mention that fact or I should I simply remove the explicit mention to dataimport.py?